rules/linux/defense_evasion_attempt_to_disable_syslog_service.toml

```
@@ -17,10 +17,41 @@ index = ["auditbeat-*", "logs-endpoint.events.*", "endgame-*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Attempt to Disable Syslog Service"
+note = """## Triage and analysis
+
+### Investigating Attempt to Disable Syslog Service
+
+Syslog is the general standard for logging system and program messages in the Linux environment.
+This service constitutes the system log service daemon, where any program can utilise its functionality for logging
+debug, security, normal operation activities in addition to the Linux kernel messages. This rule identifies the
+execution of commands to disable the syslog service. Attackers may attempt to disrupt event logging to evade security
+controls for process that use this service daemon for logs.
```

I'm not sure if this is right. Mainly flagging this in case the original version was missing a word or needed other changes. 

```suggestion
controls for processes that use this service daemon for logs.
```